src/librustc_resolve/lib.rs

```diff
@@ -257,7 +257,7 @@ fn resolve_struct_error<'b, 'a: 'b, 'c>(resolver: &'b Resolver<'a>,
                              "type `{}` is not a member of trait `{}`",
                              type_,
                              trait_);
-            err.span_label(span, &format!("not a member of trait `Foo`"));
+            err.span_label(span, &format!("not a member of `{}`", trait_));
```

Comment:
I agree being consistent is preferred.

The reason I say we might want to keep `trait` in is because, in general, we want to shoot for labels that can be read and understood on their own without having to read the error title.  They're one of the first things people see, so if we can give them enough information to fix the bug, they'll be able to fix it that much faster.

In this case, saying `trait` in the label will jog their memory of "oh wait, this must have something to do with traits".  For me, without it, I had to read the label a couple times to get to that point.

We may want also update the other method error, too...